receiver/kubeletstatsreceiver/internal/metadata/metrics.go

```diff
@@ -22,8 +22,6 @@ type RecordIntDataPointFunc func(*MetricsBuilder, pcommon.Timestamp, int64)
 
 type RecordIntDataPointWithDirectionFunc func(*MetricsBuilder, pcommon.Timestamp, int64, string, AttributeDirection)
```

Comment:
This one should be updated as well?

Reply:
I thought about it, but there's already a type `RecordIntDataPointFunc` so I decided to leave it